occupancy-quickstart/src/appSettings.json

```diff
@@ -1,7 +1,6 @@
 {
     "AADInstance": "https://login.microsoftonline.com/",
     "ClientId": "<Azure Active Directory App Id>",
```

Comment:
I like the idea of simplifying the `appSettings.json` to only values that will be altered over the course of tutorial. Moving any purely constant variables into the dotnet code is also a great idea - fewer mistakes. Quick question here, if we're trying to provide some kind of override option for the application ID, wouldn't we want to  do the same for the tenant ID too? I can see that approach being really helpful - a user could use the same local test instance to quickly switch between one or more of the web instances - they can hardcode one into dotnet and then override it with the appsettings.

Reply:
The clientId is an AAD objectId and the tenantId is the directoryId.  Both are custom to the user (since not everyone shares the same directory).

I wonder if we should rename these...  I thought these came from one of the adal libraries but I can't find it at the moment.  I assume we would have to change documentation if we rename these...

Reply:
Thanks for further clarifying the variables. LGTM.

Reply:
(We talked offline and decided not to rename these atm)